### PR TITLE
Fix port configuration UX

### PR DESCRIPTION
**Pull request by martindekov**
Fixing port configuration UX as per request from @reitermarkus
now the port is not added through environmental variable, but rather
through logic in the broker host, as the original implementation
in the multiple brokers commit

Signed-off-by: Martin Dekov <mvdekov@gmail.com>

### CODE REVIEW

**Comment by reitermarkus** [comment]
Thanks! 👍

File: README.md [hunk 1]
```diff
@@ -209,5 +209,4 @@ This configuration can be set in the YAML files for Kubernetes or Swarm.
 | `print_response`        | Default is `true` - this will output information about the response of calling a function in the logs, including the HTTP status, topic that triggered invocation, the function name, and the length of the response body in bytes |
 | `print_response_body`   | Default is `true` - this will print the body of the response of calling a function to stdout |
 | `topic_delimiter`   | Default is `,` - Specifies character upon which to split the `topic` annotation when subscribing a function to mulitple topics |
-| `custom_port` | Default is `9092` - you can set custom port in case you deployed your kafka on non-default port |
```
Comment (LucasRoesler) [inline README.md]: We should probably update the description of `broker_host` to mention that the port can be specified via this env value

**Comment by alexellis** [comment]
Tbh this is an oversight of my original implementation. I think the broker host should always be a full URL including the port. I'm not keen on the magic appending of a port, for one it is unlikely to be the same port needed for each broker host. 

Sorry to make this patch take longer. How about just doing away with the append and documenting the (breaking) change?

File: README.md (hunk 1 above)
Comment (martindekov) [reply README.md]: Will do :+1:

**Comment by martindekov** [comment]
Of course Alex, no problems I will do that ^ Thank you guys for the review :100:

File: README.md (hunk 1 above)
Comment (martindekov) [reply README.md]: Done :heavy_check_mark:

**Comment by martindekov** [comment]
I removed the logic with `:` this will break for users, if there are releases this can be mentioned as I haven't added it to README.md, though augmented the explanation on the `broker_host` variable.

**Comment by alexellis** [comment]
I'll cut a point release, then can you submit a PR to the helm chart and YAML if needed?

**Comment by alexellis** [comment]
https://github.com/openfaas-incubator/kafka-connector/releases/tag/0.4.0

**Comment by martindekov** [comment]
Yeah I will do it in a couple of hours :+1:

**Pull request merged**